banyand/liaison/grpc/testdata/server_cert.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
As @kezhenxu94 mentioned, if you copy https://github.com/grpc/grpc-go/blob/master/examples/data/x509/server_cert.pem, pls add notices and license in banyandb. But they look like only for the test purpose, how about generate them by your hand

Reply:
The `server_cert.pem` and `server_key.pem` are needed in server, the `server_cert.pem` is needed in client. The license of cert and key files are Apache-2.0 License, the same as `skywalking-banyandb`, I need to add they?

Reply:
To copy source codes needs another process more than lib dependency. It's much easy to generate TLS files by referring to https://bbengfort.github.io/2017/03/secure-grpc/

Reply:
OK